worldeditadditions/utils/parse/init.lua

```diff
@@ -4,17 +4,20 @@
 -- ██      ██   ██ ██   ██      ██ ██
 -- ██      ██   ██ ██   ██ ███████ ███████
 
+-- Unified Axes Keyword Parser
+local uak_parse = dofile(worldeditadditions.modpath.."/utils/parse/axes_parser.lua")
+-- Old axis parsing functions
 local axes = dofile(worldeditadditions.modpath.."/utils/parse/axes.lua")
-local parse = dofile(worldeditadditions.modpath.."/utils/parse/axes.lua")
-local key_instance = dofile(worldeditadditions.modpath.."/utils/parse/key_instance.lua")
 
-worldeditadditions.key_instance = key_instance
-worldeditadditions.parse = parse
-
--- Old parse functions (marked for deprecation).
--- Use parse.keytable or parse.keyword instead
-worldeditadditions.parse.axes = axes.parse_axes
-worldeditadditions.parse.axis_name = axes.parse_axis_name
+worldeditadditions.parse = {
+	keyword = uak_parse.keyword,
+	keytable = uak_parse.keytable,
```

Comment:
This is probably just me, but I don't find the `keyword` and `keytable` names here very descriptive. It's difficult to tell at a glance what their function actually is without looking at the code. The `worldeditadditions.parse` namespace contains many different parsers, so one wouldn't immediately know that these are axis parsers at first glance.

Is there are a more descriptive name we can give them?

Reply:
I can't really think of any. They don't *just* parse axes, they also do `dir`s and other keywords.

Reply:
Sure. but what about `keytable`?

Reply:
Does `keytable` parse something e.g. like `front 5 x 6 y 10 left 7`? if so, it should be `direction`, and `keyword` may be better as `direction_single` perhaps?

Reply:
`direction_keyword`?